src/styles/variables.css

```diff
@@ -46,6 +46,10 @@
   /* ----- badge ----- */
   --badge-sm-height: 64px;
   --badge-normal-height: 96px;
+
+  /* ----- card ----- */
+  --card-height: 305px;
+  --card-width: 224px;
```

Comment:
dunno if we want to have the sizes hardcoded make it flexible based on container size

Reply:
Tbh I would leave at least hardcoded width here - images and whole card wouldn't be too tight then, and we could put them later inside a flex container which will prevent it from overlapping etc.

Reply:
Ok, let's leave it as is and later we'd be able to take a closer look how it looks when further sections using `Card` will be implemented 👌